draft-poidt-ccamp-actn-poi-pluggable.md

```diff
@@ -237,29 +264,184 @@ Px = DWDM (coherent pluggable) Router ports
+   multi-layer network topology.
+
+   An architecture analysis has already been carried out by the MANTRA 
+   sub-group in the OOPT – (Open Optical & Packet Transport)a Telecom
+   Infra Project (TIP) group Project Group in the MANTRA whitepaper 
+   {{MANTRA-whitepaper-IPoWDM}}.
+
+   Two different architectural options have been identified, namely:
+~~~~ ascii-art
+     - Option 1: Dual SBI management of IPoDWDM routers
+     - Option 2: Single SBI management of IPoDWDM routers
+~~~~
+
+   Both the options foresee that the packet and the optical domain are 
+   kept separate from a management perspective.
+
+
+## Option 1 - Dual SBI management of IPoWDM routers
+
+The figure {{my_figure-2}} describes the architecture of option 1.
+
+
+~~~~ ascii-art
+
+                        +----------+
+                        |   MDSC   |
+                        +--+----+--+
+                           |    |  
+  MPI interface    +-------+    +-------+
+                   |                    | 
+              +----+----+          +----+----+
+              |  P-PNC  |          |  O-PNC  |
+              +--+---+--+          +----+----+
+                 ^   ^               ^  ^  ^
+                 |   |               |  |  |
+         +-------+   +---------------|--|--|------+
+         |                           |  |  |      |  
+         |   +-----------------------+  |  +--+   |
+         |   |               +----------+     |   |
+         |   |               |                |   |
+         |   |               v                |   |
+         v   |        +--------------+        |   v
+        +-----+      /                \      +-----+
+       / P.N.1 \..../..O.N--------O.N..\..../ P.N.2 \
+       \       /    \  Optical Domain  /    \       / 
+        +-----+      \                /      +-----+
+                      +--------------+
+
+   P.N. = Packet/Optical Node (IPoWDM router) 
+   O.N. = Optical Switching DWDM Node (ROADM)
+   ROADM = Lambda/Spectrum switch
+~~~~
+{: #my_figure-2 title="Dual SBI management Scenario"}
+
+
+   The peculiarity of this option consists of the fact that both the 
+   packet SDN controller (P-PNC) and the optical SDN controller (O-PNC)
+   have access to the coherent pluggable optics on the routers. 
+   The P-PNC is the only entity allowed to configure them, while the
+   O-PNC is granted with read-only permissions to avoid database 
+   inconsistency between them. Data write access permissions are 
+   expected to be implemented on the routers to only grant 
+   configuration rights to the P-PNC.
+
+   The read-only capabilities of the O-PNC consist in the following 
+   tasks:
+     - Device discovery, poll or stream configuration, state and 
+       static capabilities.
+     - Performance monitoring, periodically poll or stream performance 
+       counters.
+     - Fault notification, received asynchronous alarm notifications.
+
+   The consequence of this split of roles is that the P-PNC exposes
```

Comment:
Agreed, there would be these consequences. This could be an architectural choice, but it is clearly only one possible approach and it also appears to not be the best approach